src/modules/featurepopup/templates/popup.html

```diff
@@ -1,6 +1,18 @@
-<div class="anol-popup {{ openingDirection }}" ng-mousedown="makeDraggable($event)">
-    <span class="anol-popup-closer glyphicon glyphicon-remove" ng-click="close()" ng-if="!sticky" ng-mousedown="$event.stopPropagation();"></span>
-    <div class="anol-popup-content">
-        <ng-transclude></ng-transclude>
+<div ng-show="isVisible">
+    <div class="anol-popup {{ openingDirection }} {{ class }}"
+         ng-mousedown="makeDraggable($event)"
+         ng-class="{ 'hide-for-mobile': altMobileFullscreen }">
+        <span class="anol-popup-closer glyphicon glyphicon-remove" ng-click="close()" ng-if="!sticky" ng-mousedown="$event.stopPropagation();"></span>
+        <div class="anol-popup-content">
+            <ng-transclude></ng-transclude>
+        </div>
+    </div>
+    <div ng-if="altMobileFullscreen">
+        <div class="anol-popup anol-popup-mobile {{ class }}">
```

Comment:
see comment above

Reply:
class will always be a string of class names that is separated by spaces.